src/components/booking/BookingCalendar.tsx

```
@@ -121,27 +90,46 @@ const BookingCalendar: React.FC<BookingCalendarProps> = ({
-          title: "Adding to Outlook",
-          description: "Opening Outlook calendar in a new tab..."
-        });
-        break;
+    // For now, we'll use a mock appointment ID. In a real app, this would be the actual appointment ID
+    const mockAppointmentId = "temp-appointment-id";
+
+    try {
+      switch (type) {
+        case 'google':
+          // Generate Google Calendar URL
+          const startDateTime = new Date(selectedDate);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)